data/json/monstergroups/eggs.json

```diff
@@ -179,5 +181,74 @@
     "name": "GROUP_EGG_FROG",
     "type": "monstergroup",
     "monsters": [ { "monster": "mon_tadpole" } ]
+  },
+  {
+    "name": "GROUP_EGG_GOOSE",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_goose_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_GOOSE_GOLD",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_goose_golden_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_HUMMINGBIRD",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_hummingbird_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_WOODPECKER",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_woodpecker_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_COOT",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_coot_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_CORMORANT",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_cormorant_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_MOORHEN",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_moorhen_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_GREBE",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_grebe_chick" } ]
+  },
+  {
+    "name": "GROUP_EGG_TOAD",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_tadpole_brown" } ]
+  },
+  {
+    "name": "GROUP_EGG_WATER_BEETLE",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_diving_larva" } ]
+  },
+  {
+    "name": "GROUP_EGG_WATER_BUG",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_water_scorpion_larva" } ]
+  },
+  {
+    "name": "GROUP_EGG_CICADA",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_cicada_nymph" } ]
+  },
+  {
+    "name": "GROUP_EGG_BUTTERFLY",
+    "type": "monstergroup",
+    "default": "mon_caterpillar",
+    "monsters": [
+      { "monster": "mon_caterpillar_giant", "freq": 25, "cost_multiplier": 1 },
```

Comment:
The _giants need a start date, if you don't want to do an explicit variant progression (small caterpillar->small butterflies->butterfly egg-> giant caterpillar)

Reply:
the idea was that giant caterpillars may start spawning early from the eggs but alright, added a 28 days starting date to see what happens